lib/chip/service.rb

```diff
@@ -32,7 +32,8 @@ def token
     private
 
     def claims_token
-      @claims_token ||= Base64.encode64("#{config.api_username}:#{config.api_user}")
+      # TODO: create instance variable for api_user & api_password
+      @claims_token ||= Base64.encode64('fake_api_user:fake_api_password')
```

Comment:
are we going to address this TODO in this PR or another?

Reply:
Planning to address it in Service class PR;